src/main/java/at/ac/fhcampuswien/fhmdb/HomeController.java

```diff
@@ -13,104 +13,160 @@
 import javafx.collections.ObservableList;
 import javafx.collections.transformation.FilteredList;
 import javafx.fxml.FXML;
-import javafx.fxml.Initializable;
 import javafx.scene.control.TextField;
 
+import java.io.IOException;
 import java.net.URL;
 import java.util.*;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
 
-public class HomeController implements Initializable {
+public class HomeController {
+
+    /**
+     * Initializes the controller class.
+     * <p>
+     * <a href="https://docs.oracle.com/javase/8/javafx/api/javafx/fxml/Initializable.html">JavaFX Initialize</a>
+     * NOTE This interface has been superseded by automatic injection of location and resources properties into the controller.
+     * FXMLLoader will now automatically call any suitably annotated no-arg initialize() method defined by the controller.
+     * It is recommended that the injection approach be used whenever possible.
+     */
```

Comment:
This comment is for the `Initializable ` interface which was removed. Also, it is also in the lines 35-42 and 77-84

Reply:
As discussed in WhatsApp, the comment should state that the class properties are used for initialization. I will remove these doc strings